### ISSUE TITLE
Feature suggestion/request

### ISSUE THREAD

**Issue by curiousier-george** [sawhney17/Safari-Extension-Logseq-Quick-Capture#1]
This is really great, thank you!

Am I correct in understanding that this is also the Chrome extension? If so, I have a small feature suggestion/request: to add an option to make Logseq Quick Capture accessible via user-definable hotkey.

**Comment by sawhney17** [comment]
No, the chrome extension exists separately, here’s the link to that one!

https://github.com/hkgnp/chrome-extension-logseq-quickcapture

 23 Nov 2022, 2:22 AM +0400, curiousier-george ***@***.***>, wrote:
	This is really great, thank you!
Am I correct in understanding that this is also the Chrome extension? If so, I have a small feature suggestion/request: to add an option to make Logseq Quick Capture accessible via user-definable hotkey.
